src/utils.js

```diff
@@ -15,9 +15,12 @@ export function getServiceName(obj) {
 	} else {
 		throw new Error(`getServiceName must be called with a string url or with a SystemJS load object that has an address`);
 	}
-	const splits = address.split('/');
 
-	return splits[splits.length - 1];
+	let urlParts = getRegex().exec(address);
+
+	const splits = urlParts[5].split('/');
+
+	return splits[0] || splits[1];
```

Comment:
When we call `.split` above, there might be a leading `/`, if there is then `splits[0]` will be empty, so we want the second.